core/src/main/java/io/questdb/cairo/VarcharTypeDriver.java

```diff
@@ -242,30 +236,28 @@ public static int getSingleMemValueByteCount(@Nullable Utf8Sequence value) {
     /**
      * Reads a UTF-8 value from a VARCHAR column.
      *
-     * @param rowNum  the row number to read
-     * @param dataMem base pointer of the data vector
      * @param auxMem  base pointer of the auxiliary vector
+     * @param dataMem base pointer of the data vector
+     * @param rowNum  the row number to read
      * @param ab      whether to return the A or B flyweight
-     * @return a Utf8Sequence representing the value at rowNum
+     * @return a Utf8Sequence representing the value at rowNum, or null if the value is null
      */
-    public static Utf8Sequence getValue(long rowNum, MemoryR dataMem, MemoryR auxMem, int ab) {
+    public static Utf8Sequence getSplitValue(MemoryR auxMem, MemoryR dataMem, long rowNum, int ab) {
```

Comment:
nit: in the `appendValue` method the order is dataMem, auxMem and here it's auxMem, dataMem. Would be nice to update the order in `appendValue` to keep things consistent.